docker/release/debian/Dockerfile

```diff
@@ -29,8 +29,7 @@ COPY README.md /app/README.md
 COPY .git /app/.git
 
 RUN cmake .. \
- && make -j $(nproc) \
- && make install
+ && make -j $(nproc)
```

Comment:
Removed because the `make install` was creating a python `dist` with duplicated `test` folders and making the unit test on managers in python to fail because of relative path. Same on fedora

Reply:
Edit: Changed strategy. The duplicated `test` folder was linked to the `cp r` recursive copy command in the `CMakeList.txt`, decided to keep the `make install` command while robustifying the python package build in `CMakeList.txt` in the end.